docs/design/coreclr/botr/mscorlib.md

```diff
@@ -1,355 +1,356 @@
-Mscorlib and Calling Into the Runtime
+Mscorlib (a.k.a. `System.Private.CoreLib`) and calling into the Runtime
```

Comment:
Why not eliminate the mscorlib term entirely (and rename the doc)? Relegate it to a brief historical mention.

Reply:
Ha, now I read your changes. But I still suggest renaming the doc.

Reply:
Not against that. If I was going to do that, I would probably also rename `mscorlib.h` and `mscorlib.cpp` as well. Not in this PR, but in a subsequent one. What is the appetite for excising the term "mscorlib" from the functional code base? Is that a longer term goal?

Reply:
We did rename the folder it was in, and no doubt some identifiers/comments. It does not seem worth special or destabilizing work such as changing `mscorlib.h` given all the other inconsistencies and irregularities in the VM code. But, that is just my 2c.

Reply:
We have done several passes of the mscorlib -> CoreLib renaming. If you want to rename what fell through the cracks, it is fine with me.

Reply:
Will change the name of the file.